homeassistant/components/zwave_js/climate.py

```diff
@@ -526,10 +506,19 @@ async def async_set_preset_mode(self, preset_mode: str) -> None:
         # Dry and Fan preset modes are deprecated as of 2023.8
         # Use Dry and Fan HVAC modes instead
         if preset_mode_value in (ThermostatMode.DRY, ThermostatMode.FAN):
-            LOGGER.warning(
-                "Dry and Fan preset modes are deprecated and will be removed in a future release. "
-                "Use the corresponding Dry and Fan HVAC modes instead"
+            async_create_issue(
+                self.hass,
+                DOMAIN,
+                f"dry_fan_presets_deprecation_{self.entity_id}",
+                is_fixable=True,
+                is_persistent=True,
```

Comment:
This can be `False` probably as you want to auto remove the issue when HA is restarted.

Reply:
No. We don't know that the issue is fixed after a restart. We can only detect an issue when the service is called. We want the user to mark it as fixed. This is our standard for service call repair issues.